pom.xml

```diff
@@ -12,7 +12,7 @@
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Pipeline: SCM Step</name>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+SCM+Step+Plugin</url>
+    <url>https://github.com/jenkinsci/workflow-scm-step-plugin</url>
```

Comment:
Trivial question - any particular reason to hard code this, instead of using `https://github.com/jenkinsci/${project.artifactId}-plugin` ? I've seen it both ways.

Reply:
Personally, I don't think it really matters. We are never going to change the artifact ID (at least I really hope not, since it causes a ton of pain for users), so I don't expect us to have to modify just the artifact ID in `url`, so it really makes no difference. If we could somehow inherit this as a property from the parent POM, then it could be nice to be able to use the property and omit `url` here, but I don't think that works.

Reply:
Went ahead and switched in ccd41d1.